kvrocks.conf

```diff
@@ -51,6 +51,14 @@ daemonize no
 
 cluster-enabled no
 
+# By default, namespaces are stored in the configuration file and won't be replicated
+# to replicas. This option allows to change this behavior, so that namespaces are also
+# propagated to slaves. Note that:
+# 1) it won't replicate the 'masterauth' to prevent breaking master/replica replication
+# 2) it will overwrite replica's namespace with master's namespace, so be careful of in-using namespaces.
+#
+# Default: no
+repl-namespace-enabled no
```

Comment:
After a glance, it seems that even it is set to no, the namespace will also be written to the storage?

Correct me if I'm wrong.

Reply:
It will not write to DB if this option is disabled, it returns in line: https://github.com/apache/kvrocks/blob/60d607b15fe663fb1791b784ece652238badc9d1/src/server/namespace.cc#L167

Reply:
Oh I see.